### PR TITLE
Relationships changed

### PR DESCRIPTION
**Pull request by Vinaykumar-sp**

### CODE REVIEW

**Review by admin-suketa — COMMENTED** [review]
Can you remove label and desc fields from the Owner and Tag relationships table. Since we are not displaying them on a graph, we really dont need them.

File: Azure Import/NodeInfo.csv [hunk 1]
```diff
@@ -1,2 +1,2 @@
-Node Name,Description,Display Name,Catalog Name,Node Type,Is Active
-AzureSystem,AZ resources in JSON format,Azure Systems,Metadata Catalog,System Graph,true
+Node Name,Catalog Name,Display Name,Description,Node Type,Is Active
+AzureSystem,Metadata Catalog,Azure Systems,AZ resources in JSON format,System Graph,TRUE
```
Comment (admin-suketa) [inline Azure Import/NodeInfo.csv]: Can you change: AZ resources in JSON format
To: Systems setup in Azure?

File: Azure Import/PropertyInfo.csv [hunk 2]
```diff
@@ -1,10 +1,10 @@
...
-AzureSystem,azureId,10,Azure Id,String,true,true,false
-AzureSystem,name,20,Name,String,false,true,false
-AzureSystem,desc,30,Description,String,false,true,false
-AzureSystem,resourceType,40,Type,String,false,true,false
-AzureSystem,resourceLocation,50,Location,String,false,true,false
-AzureSystem,lastUpdatedBy,60,Last Updated By,String,false,true,false
-AzureSystem,lastUpdatedDateTime,70,Last Updated DateTime,LocalDateTime,false,true,false
-AzureSystem,createdBy,80,Created By,String,false,true,false
-AzureSystem,createdDateTime,90,Created DateTime,LocalDateTime,false,true,false
+AzureSystem,azureId,10,Azure Id,String,TRUE,TRUE,FALSE
```
Comment (admin-suketa) [inline Azure Import/PropertyInfo.csv]: Does upper case: TRUE even work? Have you actually tried importing this file? Why was this file changed?

File: Relationships/NodeInfo.csv [hunk 3]
```diff
@@ -1,4 +1,4 @@
-Node Name,Description,Display Name,Catalog Name,Node Type,Is Active
-OwnerRelationships,This is the table where System Id and Metadata Id to Owner Id relationships are stored.,Owner Relationships,Metadata Catalog,,true
-TagRelationships,This is the table which shows which tags are connected to which System and Metadata Ids.,Tag Relationships,Metadata Catalog,,true
-GraphicalRelationships,This table shows all the Graphical Relationships.,Graphical Relationships,Metadata Catalog,,true
\ No newline at end of file
+Node Name,Catalog Name,Display Name,Description,Node Type,Is Active
```
Comment (admin-suketa) [inline Relationships/NodeInfo.csv:1]: Again, I dont think, uppercase "TRUE" works.

File: Azure Import/PropertyInfo.csv [hunk 4]
```diff
@@ -5,6 +5,6 @@ AzureSystem,desc,30,Description,String,false,true,false
 AzureSystem,resourceType,40,Type,String,false,true,false
 AzureSystem,resourceLocation,50,Location,String,false,true,false
 AzureSystem,lastUpdatedBy,60,Last Updated By,String,false,true,false
-AzureSystem,lastUpdatedDateTime,70,Last Updated DateTime,LocalDateTime,false,true,false
+AzureSystem,lastUpdateDateTime,70,Last Updated Date Time,LocalDateTime,false,true,false
```
Comment (admin-suketa) [inline Azure Import/PropertyInfo.csv]: Extra Space in Date Time?

Comment (surajpv1) [reply Azure Import/PropertyInfo.csv]: Updated
